src/components/Person/Person.jsx

```diff
@@ -1 +1,24 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {(age)
+        && <p className="Person__age">{`I am ${age}`}</p>
+      }
+
+      <p className="Person__partner">
+        {(isMarried)
+          ? `${partnerName} is my ${(sex === 'm') ? 'wife' : 'husband'}`
+          : 'I am not married'
+        }
```

Comment:
avoid nested ternary. it's hard to read. better use a variable for the first part

Reply:
![image](https://user-images.githubusercontent.com/98764641/229051972-518f7895-7b1c-4439-bd66-d1b64653ebb8.png)